.github/workflows/ci.yml

```diff
@@ -0,0 +1,44 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    defaults:
+      run:
+        working-directory: ./web
+
+    strategy:
+      matrix:
+        node-version: [16.x]
```

Comment:
isn't TW already using note17?

Reply:
TW ships Node 16 and 17. However, Node 16 is the active LTS version. See [Node releases](https://nodejs.org/en/about/releases/) to find out more details.